data/broker-features.js

```diff
@@ -0,0 +1,255 @@
+export default [
+    {
+        feature: "Produce / Exclusive",
+        stableSince: "2.2.0",
+        document: [
+            {
+                "text": "Concept",
+                "link": "concepts-clients/#access-mode",
+            },
+        ],
+    },
+    {
+        feature: "Produce / Shared",
+        stableSince: "2.2.0",
+        document: [
+            {
+                "text": "Concept",
+                "link": "concepts-clients/#access-mode",
+            },
+        ],
+    },
+    {
+        feature: "Subscription / Exclusive",
+        experimentalSince: "",
+        betaSince: "",
+        stableSince: "2.2.0",
+        document: [
+            {
+                "text": "Concept",
+                "link": "concepts-messaging/#exclusive",
+            },
+        ],
+    },
+    {
+        feature: "Subscription / Failover",
+        experimentalSince: "",
+        betaSince: "",
+        stableSince: "2.2.0",
+        document: [
+            {
+                "text": "Concept",
+                "link": "concepts-messaging/#failover",
+            },
+        ],
+    },
+    {
+        feature: "Subscription / Shared",
+        experimentalSince: "",
+        betaSince: "",
+        stableSince: "2.2.0",
+        document: [
+            {
+                "text": "Concept",
+                "link": "concepts-messaging/#shared",
+            },
+        ],
+    },
+    {
+        feature: "Subscription / KeyShared",
+        experimentalSince: "",
+        betaSince: "",
+        stableSince: "",
```

Comment:
It should be 2.9.5

Reply:
@codelipenghui It's better to use feature release version, not bug fix release?

Reply:
@Jason918 I will use this patch release but I understand your concern that from a user point of view we need a `x.0` stable version so that we're sure that all versions after that should be stable.

@codelipenghui for example, 2.9.5 is stable, but what if 2.10.0? I guess that the significant patch landed on 2.9.5 is not included in 2.10.0.

Anyway, we can use a pattern like `2.9.5/2.10.<n>/2.11.0 or later`.

Reply:
Yes, it should be 2.10.4 for the Key_Shared subscription.